oak_attestation_verification/src/claims.rs

```diff
@@ -118,12 +120,12 @@ pub struct ClaimEvidence {
 }
 
 #[derive(Debug, Deserialize, PartialEq, Serialize)]
-pub struct EndorsementStatement {}
+pub struct Inner {}
```

Comment:
Could you add a doc to this type, especially since it is public, and not very clear what its purpose is.

Reply:
Renamed to Claimless since the struct is empty. It's a hack to support a different and non-trivial type in crate oak_ml_transparency.